packages/apps-config/src/api/spec/subspace.ts

```diff
@@ -6,27 +6,23 @@
 
 import type { Observable } from 'rxjs';
 import type { ApiInterfaceRx } from '@polkadot/api/types';
-import type { Bytes, Struct, U8aFixed, u64 } from '@polkadot/types';
+import type { Struct, u64 } from '@polkadot/types';
 import type { OverrideBundleDefinition, Registry } from '@polkadot/types/types';
 
 import { combineLatest, map } from 'rxjs';
 
 import { bestNumber, bestNumberFinalized, bestNumberLag, getBlock, subscribeNewBlocks } from '@polkadot/api-derive/chain';
 import { memo } from '@polkadot/api-derive/util';
-import { Digest, Header } from '@polkadot/types/interfaces';
+import { AccountId32, Digest, Header } from '@polkadot/types/interfaces';
 
-type FarmerPublicKey = U8aFixed
+type FarmerPublicKey = AccountId32
 
 interface HeaderExtended extends Header {
   readonly author: FarmerPublicKey | undefined;
```

Comment:
Is this Subspace header? It doesn't have `author` field, does it?

Reply:
Yes, we don't have the author field on our header, in this spec file we are manually adding this field using a custom derive method for headers related queries. This is because the front end uses a Context component that subscribes to new headers using **api.derive.chain** We are overriding that.

https://github.com/subspace/polkadot-js-apps/blob/fa82c154448c4bafbf69b1ded7556631950de887/packages/react-query/src/BlockAuthors.tsx#L58

Reply:
Got it, why is it optional though?

Reply:
I was following the original HeaderExtended type, in there the author field is optional, but now that you mention in our chain it will never be undefined. That's the idea of this override. I will update that.

Reply:
Hm, I just though about it and I think for genesis block it IS undefined

Reply:
I will leave it undefined but i will keet only the accountId definitions, removing the FarmerPublicKey declarations